homeassistant/components/music_assistant/strings.json

```diff
@@ -0,0 +1,57 @@
+{
+  "issues": {
+    "invalid_server_version": {
+      "title": "The Music Assistant server is not the correct version",
+      "description": "Check if there are updates available for the Music Assistant Server and/or integration."
+    }
+  },
+  "config": {
+    "step": {
+      "init": {
+        "data": {
+          "url": "URL of the Music Assistant server"
+        }
+      },
+      "manual": {
+        "title": "Manually add Music Assistant Server",
+        "description": "Enter the URL to your already running Music Assistant Server. If you do not have the Music Assistant Server running, you should install it first.",
+        "data": {
+          "url": "URL of the Music Assistant server"
+        }
+      },
+      "discovery_confirm": {
+        "description": "Do you want to add the Music Assistant Server `{url}` to Home Assistant?",
+        "title": "Discovered Music Assistant Server",
+        "data": {}
+      },
+      "reconfigure": {
+        "description": "Update your Music Assistant Server URL",
+        "data": {
+          "url": "URL of the Music Assistant server"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_server_version": "The Music Assistant server is not the correct version",
```

Comment:
I am missing a bit of context in this error. What should the user do to fix this? What is a correct version?

Reply:
See issues --> invalid_server_version